tests/test_models/test_localizers.py

```diff
@@ -165,3 +166,276 @@ def test_post_processing():
+            dropout_ratio=0.,
+            in_channels_activity=2048,
+            in_channels_complete=6144,
+            num_classes=20,
+            consensus=dict(
+                type='STPPTest', with_context=False, stpp_stage=(1, 1, 1)),
+            with_regression=True),
+        test_cfg=test_cfg)
+
+    imgs = torch.rand(1, 1056, 3, 224, 224)
+    relative_proposal_list = torch.Tensor([[[0.7339, 0.8749], [0.7332, 0.8921],
+                                            [0.7339, 0.9688], [0.7095, 0.8968],
+                                            [0.7313, 0.9118], [0.7379, 0.8523],
+                                            [0.7302, 0.9915], [0.6887,
+                                                               0.8749]]])
+    scale_factor_list = torch.Tensor([[[1.0000, 1.0000], [1.0000, 1.0000],
+                                       [1.0000, 0.2661], [1.0000, 1.0000],
+                                       [1.0000, 0.9773], [1.0000, 1.0000],
+                                       [1.0000, 0.0652], [1.0000, 1.0000]]])
+    proposal_tick_list = torch.LongTensor([[[700, 775, 923, 998],
+                                            [690, 774, 942, 1025],
+                                            [651, 775, 1023, 1056],
+                                            [650, 749, 947, 1045],
+                                            [676, 772, 962, 1056],
+                                            [718, 779, 900, 960],
+                                            [633, 771, 1047, 1056],
+                                            [628, 727, 923, 1022]]])
+    reg_norm_consts = torch.Tensor([[[-0.0603, 0.0325], [0.0752, 0.1596]]])
+
+    if torch.cuda.is_available():
+        localizer_ssn = build_localizer(model_cfg).cuda()
+        with torch.no_grad():
+            imgs = imgs.cuda()
+            relative_proposal_list = relative_proposal_list.cuda()
+            scale_factor_list = scale_factor_list.cuda()
+            proposal_tick_list = proposal_tick_list.cuda()
+            reg_norm_consts = reg_norm_consts.cuda()
+            # Test normal case
+            localizer_ssn(
+                imgs,
+                relative_proposal_list=relative_proposal_list,
+                scale_factor_list=scale_factor_list,
+                proposal_tick_list=proposal_tick_list,
+                reg_norm_consts=reg_norm_consts,
+                return_loss=False)
+
+            # Test SSN model without dropout
+            localizer_ssn.dropout_ratio = 0
+            localizer_ssn(
+                imgs,
+                relative_proposal_list=relative_proposal_list,
+                scale_factor_list=scale_factor_list,
+                proposal_tick_list=proposal_tick_list,
+                reg_norm_consts=reg_norm_consts,
+                return_loss=False)
+            localizer_ssn.dropout_ratio = 0.8
+
+            # Test SSN model with max spatial pooling
+            localizer_ssn.spatial_type = 'max'
```

Comment:
useless. `self.pool` is unchanged. I suggest constructing a `base_cfg` for a basic SSN model. To rebuild another SSN model, just `.copy()` and change the key to build.